build-conf/REXX.properties

```diff
@@ -0,0 +1,56 @@
+# Releng properties used by language/REXX.groovy
+
+#
+# Comma separated list of required build properties for REXX.groovy
+REXX_requiredBuildProperties=REXX_srcPDS,REXX_objPDS,REXX_loadPDS,\
+  REXX_compiler,REXX_linkEditor,REXX_tempOptions,applicationOutputsCollectionName,\
+  SCEELKED,SFANLMD
+
+#
+# REXX compiler name
+REXX_compiler=REXXCOMP
+
+#
+# linker name
+REXX_linkEditor=IEWBLINK
+
+#
+# REXX source data sets
+REXX_srcPDS=${hlq}.REXX
+REXX_objPDS=${hlq}.OBJ
+
+#
+# REXX load data sets
+REXX_REXXLoadPDS=${hlq}.REXXLOAD
```

Comment:
It seems to be confusing to use load twice. Maybe change to rexx_cexecPDS  ? 

Also aligns with the FANCMC proc:
https://www.ibm.com/docs/en/zos/2.1.0?topic=by-rexxc-fancmc

Reply:
Changed to cexecPDS